CHANGELOG.md

```diff
@@ -97,6 +97,8 @@ and this project adheres to
 
 - Fix auto enrollment with closed and specific course runs
 - Fix demo-dev command synchronization error.
+- Speedup backend tests by removing unnecessary data generations
+- Use MD5PasswordHasher in tests to increase speed
```

Comment:
Usually we add an entry into the Changelog only when changes relies on business logic.

Reply:
To be honest, I edited the changelog only to se that :smile: 
![image](https://github.com/openfun/joanie/assets/720491/f58dfc53-b812-4f34-90e6-6775acc76b13)
I'll remove them